docs/commands/ft.create.md

```diff
@@ -51,10 +51,12 @@ after the SCHEMA keyword, declares which fields to index:
 
  - `NUMERIC` - Allows numeric range queries against the value in this attribute. See [query syntax docs](/redisearch/reference/query_syntax) for details on how to use numeric ranges.
 
- - `GEO` - Allows geographic range queries against the value in this attribute. The value of the attribute must be a string containing a longitude (first) and latitude separated by a comma.
+ - `GEO` - Allows radius range queries against the value (point) in this attribute. The value of the attribute must be a string containing a longitude (first) and latitude separated by a comma.
 
  - `VECTOR` - Allows vector similarity queries against the value in this attribute. For more information, see [Vector Fields](/redisearch/reference/vectors).
 
+ - `GEOMETRY`- Allows polygon queries against the value in this attribute. The value of the attribute must follow [WKT notation](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) a list of points representing the polygon edges `POLYGON((x1 y1, x2 y2, ...)` separated by a comma.
```

Comment:
```suggestion
 - `GEOMETRY`- Allows polygon queries against the value in this attribute. The value of the attribute must follow [WKT notation](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) a list of 2D points representing the polygon edges `POLYGON((x1 y1, x2 y2, ...)` separated by a comma.
```

Reply:
Current limitations for `GEOMETRY`: supports only POLYGON, does not support `SORTABLE` and JSON multi values.